lib/checks/aria/errormessage.js

```diff
@@ -1,12 +1,16 @@
+const { aria, dom } = axe.commons;
 options = Array.isArray(options) ? options : [];
 
-var attr = node.getAttribute('aria-errormessage'),
-	hasAttr = node.hasAttribute('aria-errormessage');
+const attr = node.getAttribute('aria-errormessage');
+const hasAttr = node.hasAttribute('aria-errormessage');
 
-var doc = axe.commons.dom.getRootNode(node);
+const doc = dom.getRootNode(node);
 
-function validateAttrValue() {
-	var idref = attr && doc.getElementById(attr);
+function validateAttrValue(attr) {
```

Comment:
Why is there a method here called `validateAttrValue` when there is also one in `commons.aria`? Should they be the same method, or at least have different names?

Reply:
I don't know. It was something you approved. Is it relevant to this PR?

Reply:
That PR doesn't reference the commons method, only the inline function.